### ISSUE TITLE
Configurable Dimension support 

### ISSUE THREAD

**Issue by linshu** [awslabs/collectd-cloudwatch#43]
Is there any plan to support configurable dimension, which we can configure different dimensions through a conf file or something similar?

**Comment by mindstorms6** [comment]
Does #37 do what you're looking for?

**Comment by linshu** [comment]
I want a fully configuration dimension schema, not something hardcoded like "FixedDimension". Also, I can really add as many as possible (only limited by AWS cloudwatch) dimensions.
I put the review on the lines I want to change in #37 
Thanks.

**Comment by Poil** [comment]
You can try my old 2015 dev that was allowing that https://github.com/Poil/collectd_to_cloudwatch

**Comment by Fodoj** [comment]
@linshu I see you implemented the feature. Do you plan to make a PR to this repo?

**Comment by linshu** [comment]
@Fodoj Yes, I plan to do that. I am just doing more tests and see if anything I missed. If you think it is good, I can do the PR immediately. Thanks.

**Comment by linshu** [comment]
@Fodoj I opened a PR for this issue. Please take a look at it. Thanks.

**Comment by linshu** [comment]
What's update for this issue? Anything else are we waiting for from my side?
